package.json

```diff
@@ -19,6 +19,7 @@
     "install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files",
     "distclean": "rm -rf node_modules && yarn cache clean",
     "build": "yarn install-if-deps-outdated && yarn build-client",
+    "build-blocks": "node ./bin/build-blocks.js",
```

Comment:
What do you think about a single `build-block` command that would allow us to build a single block instead of all of them? For example we would want to build the one we're changing only, and not all of them every time.

Reply:
maybe this is a good optimization for later? what's the benefit? a slightly shorter built time?

Reply:
Yup, that's definitely good for later for when we build a ton of blocks this way, and it would make a difference.